scipy/stats/_stats_py.py

```diff
@@ -10063,7 +10060,7 @@ def quantile_test(x, *, q=0, p=0.5, alternative='two-sided'):
     expect the null hypothesis to be rejected.
 
     >>> stats.quantile_test(rvs, q=0.5, p=0.5, alternative='greater')
-    QuantileTestResult(statistic=67, statistic_sign=1, pvalue=0.9997956114162866)
+    QuantileTestResult(statistic1=67, statistic_type=1, pvalue=0.9997956114162866)
```

Comment:
@steppi isn't this supposed to be `statistic`, not `statistic1`?

Reply:
Yes